### PR TITLE
[BUG-15513] Wrong logic with image thumbnail when using Focal Point

### PR DESCRIPTION
**Pull request by bitfactory-michel-lubbers**
I have read the CLA Document and I hereby sign the CLA

## Changes in this pull request  
Resolves #
Fix for Focal point in thumbnails

## Additional info
- Bug fix: `11.2`

### CODE REVIEW

File: lib/Tool.php [hunk 1]
```diff
@@ -71,7 +71,6 @@ public static function hasCurrentRequest(): bool
      *
      * @static
      *
```
Comment (bitfactory-michel-lubbers) [inline lib/Tool.php]:
```suggestion
     *
     * @param ?string $language
```

**Comment by bitfactory-michel-lubbers** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by lukmzig** [comment]
Hi @bitfactory-michel-lubbers, thank you very much for your fix! As this is a bugfix, could you please re-base your PR on 11.3? Thank you :)

**Comment by kingjia90** [comment]
This PR got reverted due to a potential [regression](https://github.com/pimcore/pimcore/issues/17590), 
something doesn't feel right, if the change in this PR would be valid, they could be moved to https://github.com/pimcore/pimcore/blob/15af3bbf97862be757d177e9d0954fb2dcabf4de/lib/Image/Adapter.php#L195-L198

@bitfactory-michel-lubbers could you please double-check and provide an alternative PR in case? TIA

**Comment by kingjia90** [comment]
More specifically, this 
https://github.com/pimcore/pimcore/blob/423459d838b350b5bb53ea93ac8c5005aded6b7f/doc/04_Assets/03_Working_with_Thumbnails/01_Image_Thumbnails.md?plain=1#L69
`The image is resized so that it completely covers the given dimensions`, suggests that the `scaleByWidth` and `scaleByHeight` is mandatory, for example in the case when image is smaller than the given dimensions, regardless of which orientation or focal point.

**Pull request merged**